### ISSUE TITLE
[config:export:content:type] is not exporting content type

### ISSUE THREAD

**Issue by naveenvalecha** [hechoendrupal/DrupalConsole#1663]
Drupal version - 8.0.2
Drupal console - 0.10.2
drupal.phar alias - drupal

Steps to Reproduce :

 - Create a content type abc
 - drupal config:export:content:type --module=ac
 - It will ask for the content type name and whether that configuration be optional or not.but the configuration will not be written to that module.
 - console showed the message the configuration has been exported but it was not.

```
Admins-MacBook-Pro% drupal config:export:content:type --module=ac

 Content Type to be exported:
  [0] Article
  [1] cdc
  [2] Basic page
 > 1

 Export content type in module as an optional configuration (yes/no) [yes]:
 > no

 Exporting content type
 - modules/custom/ac/config/install/node.type.cdc.yml

```

**Comment by mnico** [comment]
A PR is added for this

**Comment by jmolivas** [comment]
Fixed with #1684
